rlberry/manager/experiment_manager.py

```diff
@@ -573,7 +573,11 @@ def eval_agents(
                     " Returning []."
                 )
                 return []
-            values.append(agent.eval(**eval_kwargs))
+            # Update eval_kwargs with n_simulation parameter
+            eval_kwargs_with_n_simulation = eval_kwargs.copy()
+            eval_kwargs_with_n_simulation["n_simulation"] = 1
+
+            values.append(agent.eval(**eval_kwargs_with_n_simulation))
```

Comment:
Thanks, LGTM! 
Maybe a small comment though: could you add a warning (with `logger.info`) so that when `n_simulation` already exists in  `eval_kwargs`, we have a warning saying that it is overwritten. 
eval_kwargs is not meant to be used like that but if people do it, let us give a warning that we are overwriting with 1.

Reply:
I added the warning, thank you for the help. Is it looking good now?

Reply:
Thanks. There are still some tests that do not pass, see my comment at the end of the discussion and the "detail" button the checks at the bottom. Some test was broken by the change.

Reply:
I added the argument but some tests are still failing

Reply:
The tests are failing due to https://github.com/rlberry-py/rlberry/issues/369 , this is ok.